tests/python/relay/test_op_level1.py

```diff
@@ -426,7 +486,7 @@ def test_dense():
     for dtype in ["float16", "float32"]:
         # Dense accuracy for float16 is poor
         if dtype == "float16":
-            return
+            continue
```

Comment:
did some PR affect this?

Reply:
I think this is a bug. This function will always return, and no check will be taken in this UT.